kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -64,17 +65,18 @@
    *                           consist of both a name and a value.
    * @return The result BigQuery row content.
    */
-  public Map<String, Object> convertRecord(SinkRecord kafkaConnectRecord) {
+  public Map<String, Object> convertRecord(SinkRecord kafkaConnectRecord, Boolean shouldConvertSpecialDouble) {
     Schema kafkaConnectSchema = kafkaConnectRecord.valueSchema();
     if (kafkaConnectSchema.type() != Schema.Type.STRUCT) {
       throw new
           ConversionConnectException("Top-level Kafka Connect schema must be of type 'struct'");
     }
-    return convertStruct(kafkaConnectRecord.value(), kafkaConnectSchema);
+    return convertStruct(kafkaConnectRecord.value(), kafkaConnectSchema, shouldConvertSpecialDouble);
   }
 
   @SuppressWarnings("unchecked")
-  private Object convertObject(Object kafkaConnectObject, Schema kafkaConnectSchema) {
+  private Object convertObject(Object kafkaConnectObject, Schema kafkaConnectSchema,
+                               Boolean shouldConvertSpecialDouble) {
```

Comment:
Use `boolean` primitive instead of boxed `Boolean` class. Same for below usages.

Note: I acknowledge that these params will go away when the param is moved to record converter.